samples/C/asm.h

```diff
@@ -0,0 +1,27 @@
+/* CarbonOS System/Kernel
+ * Copyright 2015-2017 David Aylaian
+ * Licensed under GitHub CLA: https://cla.github.com/agreement
```

Comment:
I also noticed that the project the code was originally authored for is [licensed under the Apache 2.0 License](https://github.com/DavidAylaian/CarbonOS/blob/6dbd6ec/LICENSE.md). Which, [as you'll see](https://github.com/github/linguist/blob/master/vendor/licenses/config.yml#L2), is an acceptably permissive license. =)

In other words, you didn't have to relicense anything. ;-)